scripts/create-release-tars.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
Why not make this a leeway package?

With a `generic` package and `leeway build --save` you'd basically get this for free.